src/commands/validate.rs

```diff
@@ -23,14 +23,18 @@ pub struct Args {
 
     /// The name of the task or workflow to validate inputs against.
     ///
-    /// If inputs are provided, this will be attempted to be inferred from the
-    /// prefixed names of the inputs (e.g, `<name>.<input-name>`).
+    /// This argument is required if trying to validate a task or workflow
+    /// without any inputs.
     ///
-    /// If no inputs are provided and this argument is not provided, it will be
-    /// assumed you're trying to validate the workflow present in the specified
-    /// document.
+    /// If `entrypoint` is not specified, all inputs (from both files and
+    /// key-value pairs) are expected to be prefixed with the name of the
+    /// workflow or task being run.
+    ///
+    /// If `entrypoint` is specified, it will be prefixed (with a `.` delimiter)
+    /// to all key-value pair inputs on the command line. Keys specified within
+    /// files are unchanged by this argument.
```

Comment:
Whatever wording is desirable for `run`'s `entrypoint` documentation should be mirrored here.